src/app/families/families.ts

```diff
@@ -245,8 +247,20 @@ export class Families extends IdEntity {
       fd.deliveryComments.value = settings.comment;
       fd.distributionCenter.value = settings.distCenter;
       fd.courier.value = settings.courier;
+      if (settings.deliverStatus) fd.deliverStatus.value = settings.deliverStatus;
+      if (settings.archive) fd.archive.value = settings.archive;
       if (settings.selfPickup)
         fd.deliverStatus.value = DeliveryStatus.SelfPickup;
+
+      '\nsettings.basketType: ' + settings.basketType, 
```

Comment:
?????

Reply:
Some logging for this interface mostly for lab manager integration. We can remove later when the systems are stable.

Reply:
very strange. this code worked locally and on mlt-test before I created the PR. 
fixed now and verifying.